tests/pos/i13044.scala

```diff
@@ -1,4 +1,4 @@
-//> using options -Xmax-inlines:33
+//> using options -Xmax-inlines:35
```

Comment:
Change required when backported to 3.5.0 but not needed in main

Reply:
Yes the same change was needed in 2facda005615361a701616adccd2d77fb5bea1d1